http/swagger.yml

```diff
@@ -2361,7 +2361,7 @@ paths:
           required: true
           description: The ID of the dashboard to update.
       responses:
-        '200':
+        '201':
```

Comment:
Did we only make the created items with a 201 code? Seems this is an updated one.